appservice/src/intTest/HibernateStatsTest.md

```diff
@@ -2,30 +2,22 @@
 
 | Operation| Close statement| Collection fetch| Collection load| Collection recreate| Collection remove| Collection update| Connect| Entity delete| Entity fetch| Entity insert| Entity load| Entity update| Flush| Optimistic failure| Prepare statement| Query cache hit| Query cache miss| Query cache put| Query execution| Query execution max time| Second level cache hit| Second level cache miss| Second level cache put| Session close| Session open| Successful transaction| Transaction
 | ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---| ---
-| GetUserWithoutPrivateDataFirst| 0| 0| 0| 0| 0| 0| 1| 0| 0| 0| 1| 0| 1| 0| 1| 0| 0| 0| 0| 0| 0| 0| 0| 3| 3| 1| 1
-| GetUserWithoutPrivateDataSecond| 0| 0| 0| 0| 0| 0| 1| 0| 0| 0| 1| 0| 1| 0| 1| 0| 0| 0| 0| 0| 0| 0| 0| 2| 2| 1| 1
-| GetUserWithPrivateDataFirst| 0| 7| 15| 0| 0| 0| 1| 0| 3| 0| 66| 0| 3| 0| 23| 0| 0| 0| 10| 5| 0| 0| 0| 2| 2| 3| 3
-| GetUserWithPrivateDataSecond| 0| 1| 9| 0| 0| 0| 1| 0| 3| 0| 63| 0| 3| 0| 20| 0| 0| 0| 10| 6| 0| 0| 0| 2| 2| 3| 3
-| GetMessagesFirst| 0| 1| 9| 0| 0| 0| 1| 0| 8| 0| 128| 0| 3| 0| 27| 0| 0| 0| 12| 39| 0| 0| 0| 2| 2| 5| 5
-| GetMessagesSecond| 0| 1| 9| 0| 0| 0| 1| 0| 8| 0| 128| 0| 3| 0| 27| 0| 0| 0| 12| 46| 0| 0| 0| 2| 2| 5| 5
-| GetDayActivityOverviewsFirst| 0| 4| 11| 0| 0| 0| 1| 0| 2| 0| 40| 0| 2| 0| 15| 0| 0| 0| 6| 5| 0| 0| 0| 2| 2| 2| 2
-| GetDayActivityOverviewsSecond| 0| 1| 9| 0| 0| 0| 1| 0| 3| 0| 38| 0| 2| 0| 11| 0| 0| 0| 5| 9| 0| 0| 0| 2| 2| 2| 2
-| GetWeekActivityOverviewsFirst| 0| 6| 22| 0| 0| 0| 1| 0| 2| 0| 79| 0| 2| 0| 17| 0| 0| 0| 6| 6| 0| 0| 0| 2| 2| 2| 2
-| GetWeekActivityOverviewsSecond| 0| 3| 19| 0| 0| 0| 1| 0| 3| 0| 76| 0| 2| 0| 13| 0| 0| 0| 5| 7| 0| 0| 0| 2| 2| 2| 2
-| GetDayActivityOverviewsWithBuddiesFirst| 0| 7| 51| 0| 0| 0| 1| 0| 3| 0| 259| 0| 3| 0| 32| 0| 0| 0| 16| 6| 0| 0| 0| 2| 2| 3| 3
-| GetDayActivityOverviewsWithBuddiesSecond| 0| 5| 49| 0| 0| 0| 1| 0| 3| 0| 259| 0| 3| 0| 30| 0| 0| 0| 16| 5| 0| 0| 0| 2| 2| 3| 3
-| GetDayActivityDetailsLastFridayFirst| 0| 4| 9| 0| 0| 0| 1| 0| 2| 0| 33| 0| 2| 0| 15| 0| 0| 0| 6| 5| 0| 0| 0| 2| 2| 2| 2
-| GetDayActivityDetailsLastFridaySecond| 0| 1| 6| 0| 0| 0| 1| 0| 2| 0| 27| 0| 2| 0| 10| 0| 0| 0| 5| 5| 0| 0| 0| 2| 2| 2| 2
+| GetUserWithPrivateDataFirst| 0| 1| 9| 0| 0| 0| 1| 0| 3| 0| 63| 0| 3| 0| 20| 0| 0| 0| 10| 6| 0| 0| 0| 2| 2| 3| 3
+| GetUserWithPrivateDataSecond| 0| 1| 9| 0| 0| 0| 1| 0| 3| 0| 63| 0| 3| 0| 20| 0| 0| 0| 10| 5| 0| 0| 0| 2| 2| 3| 3
+| GetMessagesFirst| 0| 1| 9| 0| 0| 0| 1| 0| 8| 0| 128| 0| 3| 0| 27| 0| 0| 0| 12| 54| 0| 0| 0| 2| 2| 5| 5
+| GetMessagesSecond| 0| 1| 9| 0| 0| 0| 1| 0| 8| 0| 128| 0| 3| 0| 27| 0| 0| 0| 12| 38| 0| 0| 0| 2| 2| 5| 5
+| GetDayActivityOverviewsFirst| 0| 4| 14| 0| 0| 0| 1| 0| 3| 0| 78| 0| 2| 0| 24| 0| 0| 0| 11| 5| 0| 0| 0| 2| 2| 2| 2
```

Comment:
It's not OK, but this isn't caused by this PR but by earlier ones. We lost sight of this and didn't run it since September last year. Since then, we introduced these performance degradations. Now I stumbled upon this test and noticed it didn't run anymore. I made it work and regenerated the data and the outcome is quite bad. I've created [YD-665](https://jira.yona.nu/browse/YD-665).

Reply:
Have you checked that this PR itself does not cause degradations?

Reply:
I don't know how I could, but I'm convinced it doesn't, as this PR does not need any extra data. It just uses the user or buddy entity that is anyway already loaded.